backend/tests/v5/e2e/services/chat/modelEvents/modelUploads.test.js

```diff
@@ -221,6 +225,103 @@ const queueFinishedTest = () => {
 	});
 };
 
+const revisionAddedTest = () => {
```

Comment:
as discussed, you shouldn't need an additional test for this? `queueFinishedTest` essentially is the same situation